server/methods/core/orders.js

```diff
@@ -290,10 +300,13 @@ export const methods = {
       ordersInventoryAdjust(order._id);
     }
 
+    const billingRecord = order.billing.find((billing) => { return billing.shopId === Reaction.getShopId(); });
+    const shippingRecord = order.shipping.find((shipping) => { return shipping.shopId === Reaction.getShopId(); });
```

Comment:
on a single line, you can use the shorthand find if you prefer.
```
order.shipping.find((shipping) => shipping.shopId === Reaction.getShopId());
```
Just trying to avoid using the shorthand for multi-line functions.